packages/js/data/src/product-form/resolvers.ts

```diff
@@ -34,10 +29,6 @@ export function* getFields() {
 	}
 }
 
-export function* getCountry() {
-	yield resolveSelect( STORE_NAME, 'getProductForm' );
-}
```

Comment:
Are these not being used anymore? or was this added by accident?

Reply:
It looks like it was just by accident that I included this when I originally created the data store.